assets/component-card.css

```diff
@@ -214,16 +214,6 @@
   }
 }
 
-.card-information quantity-popover volume-pricing {
-  margin-top: 4.2rem;
-}
-
-@media screen and (min-width: 990px) {
-  .card-information quantity-popover volume-pricing {
```

Comment:
```
.card-information quantity-popover div:not(quantity__rules) ~ volume-pricing {
}
```

Lets re-add this but we only want to apply it for vol pricing that dont have qty rules before